app/_hub/kong-inc/acme/index.md

```diff
@@ -16,12 +16,14 @@ categories:
 kong_version_compatibility:
     community_edition:
       compatible:
+        - 2.4.x
         - 2.3.x
         - 2.2.x
         - 2.1.x
         - 2.0.x
     enterprise_edition:
       compatible:
+       - 2.4.x
```

Comment:
```suggestion
        - 2.4.x
```
I think that's the right level, hard to tell here - just make sure it aligns with the others, right now it's preventing the section from displaying for this plugin.